src/tools/compiletest/src/header.rs

```diff
@@ -1034,19 +1034,6 @@ impl Config {
         }
     }
 
-    pub fn find_rust_src_root(&self) -> Option<PathBuf> {
-        let mut path = self.src_base.clone();
-        let path_postfix = Path::new("src/etc/lldb_batchmode.py");
-
-        while path.pop() {
-            if path.join(&path_postfix).is_file() {
-                return Some(path);
-            }
-        }
-
-        None
-    }
-
```

Comment:
This was semi-necessary back in https://github.com/rust-lang/rust/pull/16322 because bootstrap didn't exist and compiletest was invoked directly, but now we have bootstrap so this isn't needed anymore.